src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/group_convolution.cpp

```diff
@@ -313,6 +343,39 @@ const std::vector<fusingSpecificParams> fusingParamsSetBF16{emptyFusingSpec,
                                                             // sum
                                                             fusingSum};
 
+const std::vector<fusingSpecificParams> fusingParamsSet_Brdgmm{emptyFusingSpec,
+                                                               // eltwise
+                                                               fusingRelu,
+                                                               fusingPRelu1D,
+                                                               // depthwise
+                                                               fusingReluScaleShift,
+                                                               // fake quantize
+                                                               fusingFakeQuantizePerTensorRelu,
+                                                               fusingFakeQuantizePerChannelRelu
+                                                               // sum
+                                                               //fusingSumEluFQ,
+                                                               //fusingSum
```

Comment:
Any reason why some items a commented out?

Reply:
about debugging the commented item, I have already put my finding in the ticket. help to have a look , thanks

Reply:
Please add the comment why Sum post ops test cases are disabled

Reply:
updated